platform-mbean/src/main/java/org/jboss/as/platform/mbean/RuntimeMXBeanAttributeHandler.java

```diff
@@ -116,7 +116,11 @@ static void storeResult(final String name, final ModelNode store) {
         } else if (PlatformMBeanConstants.BOOT_CLASS_PATH_SUPPORTED.equals(name)) {
             store.set(ManagementFactory.getRuntimeMXBean().isBootClassPathSupported());
         } else if (PlatformMBeanConstants.BOOT_CLASS_PATH.equals(name)) {
-            store.set(ManagementFactory.getRuntimeMXBean().getBootClassPath());
+            if (!ManagementFactory.getRuntimeMXBean().isBootClassPathSupported()) {
+                store.set(""); // JDK9 and above does not have boot class path
+            } else {
+                store.set(ManagementFactory.getRuntimeMXBean().getBootClassPath());
+            }
```

Comment:
without this change we have mbean tests failures, I think it was the test that tries to read whole model via jmx.
But yes it should have thrown UnsupportedOperationException and we would catch it but it doesn't. 
I am still convinced this is bug in JDK itself and as such am not really happy getting this "fix/hack" into master just yet.